packages/emotion/src/styleUtils/index.js

```diff
@@ -82,7 +83,14 @@ const useTheme = () => {
 const useStyle = (componentName, generateStyle, props, ...rest) => {
   const theme = useTheme()
 
-  return makeStyles(theme, componentName, generateStyle, props)(...rest)
+  const styles = calculateStyles(
+    theme,
+    componentName,
+    generateStyle,
+    props
+  )(...rest)
+
+  return polyfillStyles(styles)
```

Comment:
This returns sometimes undefined (bidirectional.js line 152), if this is expected it should be mentioned in the description.

Reply:
answered above